### ISSUE TITLE
Please note.

### ISSUE THREAD

**Issue by NullArray** [NullArray/Shellware#1]
This project hasn't been updated in a while, as such, syntax errors may be present due to the fact that some of the libraries used in this project have been updated and changed since the initial release of Shellware.

Currently i don't have an update planned for this program. Although i might release a similar app in the future. One that is more modern and feature rich.

Furthermore, there are a number of malware related projects that i've been working on for a while. I don't have an ETA on those, but what i can say is that they will be far more sophisticated than this project.

If you do intend to test this project or use it to one end or another feel free to open a ticket if you're having any issues and i'll try to assist you to the best of my abilities. 

TL;DR My focus at the time of writing this, is not on updating and/or refactoring this project's code base. For now i am still maintaining it, but minimally.
